refresh/index.js

```diff
@@ -117,7 +117,20 @@ module.exports = generators.Base.extend({
       }
 
       // Bluemix configuration
-      this.bluemix = (this.spec.bluemix === true);
+      this.bluemix = this.spec.bluemix || undefined;
+
+      if(typeof(this.bluemix) === 'object') {
+        // Validate the values
+        if((typeof(this.bluemix.host) !== 'string')) {
+            this.bluemix.host = undefined;
+        }
+        if((typeof(this.bluemix.domain) !== 'string')) {
+            this.bluemix.domain = undefined;
+        }
+        if(typeof(this.bluemix.memory) !== 'string') {
+            this.bluemix.memory = '128M';
+        }
+      }
```

Comment:
Should be an `else` here that ensures if `bluemix` is not an object then it is a boolean.